WooCommerce/Classes/ViewRelated/Products/UpdateInventory/UpdateProductInventoryViewModel.swift

```diff
@@ -1,6 +1,11 @@
 import Combine
 import Yosemite
 
+enum UpdateInventoryError: Error {
+    case nonSupportedQuantity
+    case generic
```

Comment:
I went ahead for now with these two errors, but happy to iterate further since we may need to define these more specifically depending on the case.